oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundingBox.java

```diff
@@ -75,52 +65,35 @@ public OSHDBBoundingBox(double minLon, double minLat, double maxLon, double maxL
   public OSHDBBoundingBox(int minLon, int minLat, int maxLon, int maxLat) {
     this((double) minLon, (double) minLat, (double) maxLon, (double) maxLat);
   }
-
-  public void set(OSHDBBoundingBox bbox) {
-   this.minLon = bbox.minLon;
-   this.maxLon = bbox.maxLon;
-   this.minLat = bbox.minLat;
-   this.maxLat = bbox.maxLat;
-  }
   
-  public void add(OSHDBBoundingBox bbox){
-    this.minLon = Math.min(minLon,bbox.minLon);
-    this.maxLon = Math.max(maxLon,bbox.maxLon);
-    this.minLat = Math.min(minLat,bbox.minLat);
-    this.maxLat = Math.max(maxLat,bbox.maxLat);
-  }
-
-  public double getMinLon() {
-    return minLon * OSHDB.GEOM_PRECISION;
+  public OSHDBBoundingBox add(OSHDBBoundingBox bbox){
+    return new OSHDBBoundingBox(
+        Math.min(minLon,bbox.minLon),
+        Math.min(minLat,bbox.minLat),
+        Math.max(maxLon,bbox.maxLon),
+        Math.max(maxLat,bbox.maxLat));
   }
   
   public long getMinLonLong(){
     return minLon;
   }
 
-  public double getMaxLon() {
-    return maxLon * OSHDB.GEOM_PRECISION;
-  }
-
   public long getMaxLonLong(){
     return maxLon;
   }
   
-  public double getMinLat() {
-    return minLat * OSHDB.GEOM_PRECISION;
-  }
-  
   public long getMinLatLong(){
     return minLat;
   }
 
-  public double getMaxLat() {
-    return maxLat * OSHDB.GEOM_PRECISION;
-  }
-  
   public long getMaxLatLong(){
     return maxLat;
   }
+  
+  @Override
+  public OSHDBBoundingBox getBoundingBox() {
```

Comment:
Actually it might be usefull/needed. 
The `OSHDBBoundable::getBoundingBox()` returns an instance of `OSHDBBoundingBox` and not `OSHDBBoundable`. 
So if you want to store the bboxes for example into a map it makes a different if you use `OSHDBBoundingBox` or `OSHDBBoundable`
e.g.
```
 entity a -> bbox(0,0,1,1)
 entity b -> bbox(0,0,1,1)
 boundable ba = (OSHDBBoundable) a
 boundable bb = (OSHDBBoundable) b
 
 ba.equals(bb) -> false
 a.getBoundingBox().equals(b.getBoundingBox()) -> true
```

Reply:
Ok, I see. But as we don't need it at the moment, it should be omitted for now, shouldn't it?

Also, a slightly less confusing alternative solution could be to add a copy-constructor to the `OSHDBBoundingBox` class:

```java
  public OSHDBBoundingBox(OSHDBBoundable boundable) {
    this.minLon = boundable.getMinLonLong();
    …
```

then you could do something like this:

```
 entity a -> bbox(0,0,1,1)
 OSHDBBoundingBox ba = new OSHDBoundingBox(a)
 OSHDBBoundingBox bb = …
 
 ba.equals(bb) -> true
```